packages/gatsby-source-shopify/src/constants.js

```diff
@@ -12,6 +12,7 @@ export const PRODUCT_VARIANT = `ProductVariant`
 export const PRODUCT_METAFIELD = `ProductMetafield`
 export const PRODUCT_VARIANT_METAFIELD = `ProductVariantMetafield`
 export const SHOP_POLICY = `ShopPolicy`
+export const SHOP_DETAILS = `Shop`
```

Comment:
should the name better `ShopDetail` to be equal to constant name SHOP_DETAIL and also to the other name `ShopPolicy`?

Reply:
and there is a Conflict here with line 17:

```js
export const SHOP = `shop`
```

Reply:
or they are some magic other variables?

Reply:
`shop` is used to as an array includes method and is an internal thing where as `Shop` with the uppercase `S` relates to the Shopify GraphQL query end point

Reply:
ok - did find my error 

But the Query name should better ShopDetails to be same as Constant name? otherwise its a little inconsistency because of names...

Reply:
Yeah that's a tricky one because "Shop" is used all over the place. All the things that have "details" in it aren't ever surfaced when the plugin is used so maybe it doesn't matter, it's merged anyway so 🤷‍♂️